src/address.js

```diff
@@ -118,8 +118,13 @@ export class Address {
         return Address.account(scAddress.accountId().ed25519());
       case xdr.ScAddressType.scAddressTypeContract().value:
         return Address.contract(scAddress.contractId());
-      case xdr.ScAddressType.scAddressTypeMuxedAccount().value:
-        return Address.muxedAccount(scAddress.muxedAccount());
```

Comment:
There is also a type error here; `Address.muxedAccount` accepts a `Buffer` parameter, but `scAddress.muxedAccount()` is not a `Buffer`.